.github/workflows/build-push.yml

```diff
@@ -4,7 +4,7 @@ name: Build && Push
 # events but only for the develop branch
 on:
   push:
-    branches: [master]
+    branches: [master, 1141-move-stfc-rollassignment-into-uop]
```

Comment:
Will this be reverted?

Reply:
yep needed to push a image to the repo so I could run all the tests in the core repo will revert now